.github/CONTRIBUTING.md

```diff
@@ -2,81 +2,97 @@
+  delete this section.
+-->
+
+We value all community members and aim to foster a positive environment. Please
+review our [Code of conduct](./CODE_OF_CONDUCT.md) before engaging in any
+project spaces.
 
 ## Prerequisites
 
-Before contributing to Carbon, you should make sure you have the following tools
-installed:
+Before contributing, ensure you have these tools installed:
 
-- [Node.js](https://nodejs.org/en/download/) v18 or above here or follow their
-  installation through a package manager
-  [here](https://nodejs.org/en/download/package-manager/))
-  - If you're on macOS, we recommend using
-    [`nvm`](https://github.com/nvm-sh/nvm) to help manage different versions of
-    Node.js [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) as your
-    version manager for Node.
-- Git
-- [Yarn](https://yarnpkg.com/en/docs/install)
-- [WSL](../docs/guides/setup/windows.md) (Windows Only)
+- [Node.js](https://nodejs.org/en/download/) – Use the version specified in
+  [.nvmrc](../.nvmrc).
+  - On macOS, we recommend using [nvm](https://github.com/nvm-sh/nvm) to manage
+    Node versions.
+- [Git](https://github.com/git-guides/install-git)
+- [Yarn](https://yarnpkg.com/getting-started)
+- [WSL](../docs/guides/setup/windows.md) (Windows only)
 
-If you're using Windows, there are some
-[additional steps](https://github.com/carbon-design-system/carbon/blob/main/docs/guides/setup/windows.md)
-you'll need to take.
+If you're on Windows, follow
+[these additional steps](../docs/guides/setup/windows.md).
 
-You'll also need a code editor to make changes to Carbon. There are many to
-choose from but some popular options are
-[VSCode](https://code.visualstudio.com/), [Atom](https://atom.io), and
-[Sublime](https://www.sublimetext.com/).
+You'll also need a code editor. [VS Code](https://code.visualstudio.com/) and
+[Sublime Text](https://www.sublimetext.com/) are popular options.
 
-With that all in place, you're ready to start contributing to Carbon!
+Once these tools are installed, you're ready to contribute!
 
 ## Start contributing
 
-## - Setting up your environment -
+### Setting up your environment
+
+#### Fork the repository
+
+<!--
+TODO: Should this entire section be simplified to just include a link to
+GitHub’s fork instructions? These steps are subject to change as GitHub updates
+its UI, as can be seen with the previous set of instructions being outdated. It
+would be more straightforward to just point to the official GitHub documentation
+rather than maintain a separate set of instructions.
 
-### 1. Fork the repo:
+Also, are we expecting people that have zero experience with Git or GitHub to
+contribute to Carbon? If so, it seems like a better use of our time to point the
+user to resources on Git and GitHub, like tutorials (e.g.,
+https://github.com/firstcontributions/first-contributions,
+https://github.com/git-guides, etc.), rather than re-explain basic workflows.
```

Comment:
Yes, designers often join the team with zero git experience, for example. Agree lets add those links.